core/tx_pool.go

```diff
@@ -1362,6 +1364,13 @@ func (pool *TxPool) runReorg(done chan struct{}, reset *txpoolResetRequest, dirt
 			txs = append(txs, set.Flatten()...)
 		}
 		pool.txFeed.Send(NewTxsEvent{txs})
+
+		log.Debug("runReorg", "len(txs)", len(txs))
+		if len(txs) > dumpReorgTxHashThreshold {
+			for _, txs := range txs {
```

Comment:
Will dumping these tx hashes on each run of `runReorg` be expensive? Should we do it periodically (e.g. once per minute) instead?

Reply:
We only dump when `> dumpReorgTxHashThreshold`, the frequency is unlikely to be <1min, see https://monitor.scroll.tech/d/P1C1I2qSk/broadcast-investigation?orgId=1&from=now-24h&to=now&var-env=InfluxDB-mainnet-geth&var-host=l2geth_bootnode_0&var-percentile=95&viewPanel=36&refresh=10s